packages/bot-web-ui/src/components/toolbox/toolbox-items.jsx

```diff
@@ -0,0 +1,767 @@
+import React from 'react';
+import ReactDomServer from 'react-dom/server';
+
+const Block = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('block', props);
+};
+
+const Category = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('category', props);
+};
+
+const Field = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('field', props);
+};
+
+const Mutation = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('mutation', props);
+};
+
+const Next = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('next', props);
+};
+
+const Shadow = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('shadow', props);
+};
+
+const Statement = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('statement', props);
+};
+
+const Value = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('value', props);
+};
+
+const Xml = ({ ...props }) => {
+    props.is = 'blockly';
+    return React.createElement('xml', props);
+};
+
+export const ToolboxItems = ReactDomServer.renderToStaticMarkup(
+    <Xml xmlns='http://www.w3.org/1999/xhtml'>
+        <Category
+            id='trade_parameters'
+            name='Set up your trade'
+            description='Choose the market, contract type, stake, duration, and barrier.'
```

Comment:
We can remove the `description` from each `<Category>` here as they're no longer in use.

Reply:
Ah, done 👍